packages/flutter_tools/templates/ffi_plugin/windows.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
this file should have a lot more documentation explaining what everything does, what the implications are, what the pitfalls are, how it works, etc.